ckanext/qgov/common/templates/admin/config.html

```diff
@@ -22,3 +22,23 @@
 {{ form.textarea('ckanext.data_qld.excluded_display_name_words', id='field-ckanext.data_qld.excluded_display_name_words', label=_('Excluded display name words'), placeholder=_('eg.\nAdmin\nEditor'), value=data['ckanext.data_qld.excluded_display_name_words'], error=errors['ckanext.data_qld.excluded_display_name_words']) }}
 
 {% endblock %}
+
+{% block admin_form_help %}
```

Comment:
is this from the root theme?

Reply:
Yes. We currently customise it in ckanext-data-qld, but not in ckanext-publications-qld-theme. Since ckanext-qgov makes some changes to the fields, it makes sense to customise the help text too.